Added examples from Plots.jl (#166)
JuliaPlots/MakieGallery.jl: docs/src/basic-tutorials.md

```

## Getting the latest version of Makie

> Run the following commands in the package manager, this is required if you have Julia version above v1.0.0:

Reviewer: ```suggestion
Run the following commands in the package manager, this is required if you have Julia version 1.0 or newer:
```

Reviewer 2: Makie doesn't work on 0.7

```suggestion
Run the following commands in the package manager:

> Run the following commands in the package manager, this is required if you have Julia version above v1.0.0:

Reviewer 2: Makie can't be installed on VERSION <= 0.7:

```suggestion
Run the following commands in the package manager:

The user-facing functions of Makie are pretty well documented, so you can usually use the help mode in the REPL, or your editor of choice. If you countinue to have issues, see [Getting Help](@ref).

> ## Plotting in Scripts

Reviewer: I like this section a lot. It might help people making the transition from plots to makie!

If you go and try out each example listed here in the interactive Julia terminal/repl then you would not need the _display(scene)_ at the end. However if you go and try to do it in script _.jl_ file then you
would need the _display(scene)_ to display the graph, as in the repl Julia automatically calls _display_.

> ```Julia

Reviewer 2: ```suggestion
```julia
